dotcom-rendering/src/web/components/NavList.tsx

```diff
@@ -0,0 +1,74 @@
+import { css } from '@emotion/react';
+import {
+	between,
+	body,
+	border,
+	from,
+	palette,
+	space,
+	until,
+} from '@guardian/source-foundations';
+import { Link } from '@guardian/source-react-components';
+import type { DCRContainerPalette } from 'src/types/front';
+import type { TrailType } from '../../types/trails';
+import { decideContainerOverrides } from '../lib/decideContainerOverrides';
+
+type Props = {
+	trails: TrailType[];
+	containerPalette?: DCRContainerPalette;
+};
+
+const ulStyles = css`
+	${until.tablet} {
+		column-count: 1;
+	}
+	${between.tablet.and.desktop} {
+		column-count: 3;
+	}
+	column-count: 4;
+
+	column-rule: 0.0625rem solid ${border.secondary};
```

Comment:
If we're supporting container overrides then the border colour should also be overridden too (ditto below)

Reply:
I'm not sure! I originally thought this too but when I looked at ordinary containers I noticed that their dividers don't use containerPalette for deciding colour either, im not sure if this is an oversight?

Reply:
Coolio, I'll add it here and then look at adding it to all of the containers then!

Reply:
See https://github.com/guardian/dotcom-rendering/pull/6248

Reply:
Nice 👍